### PR TITLE
adding new commit

### PR DESCRIPTION
**Pull request by GhaithAshkar**
i want to add new commit

### CODE REVIEW

File: ERP.py [hunk 1]
```diff
@@ -17,4 +17,5 @@
 
 weekly_sales = data.groupby('product_id').resample('W')['sales'].sum().reset_index()
 
-print(weekly_sales)
```
Comment (Silver2er0) [inline ERP.py:20]: don't do this

File: ERP.py [hunk 2]
```diff
@@ -17,4 +17,5 @@
...
\ No newline at end of file
+print(weekly_sales)
+# commit
```
Comment (Silver2er0) [inline ERP.py:21]: bad code

**Comment by GhaithAshkar** [comment]
ok

File: ERP.py (hunk 2 above)
Comment (GhaithAshkar) [reply ERP.py:21]: okkkkkk

**Pull request merged**